samples/contrib/aws-samples/ground_truth_pipeline_demo/mini-image-classification-pipeline.py

```diff
@@ -119,14 +115,19 @@ def ground_truth_test(region='us-west-2',
         max_concurrent_tasks=ground_truth_max_concurrent_tasks
     ).apply(use_aws_secret('aws-secret', 'AWS_ACCESS_KEY_ID', 'AWS_SECRET_ACCESS_KEY'))
 
+    channelObj['ChannelName'] = 'train'
+    channelObj['DataSource']['S3DataSource']['S3Uri'] = str(ground_truth_train.outputs['output_manifest_location'])
+    channelObjList.append(copy.deepcopy(channelObj))
+    channelObj['ChannelName'] = 'validation'
+    channelObj['DataSource']['S3DataSource']['S3Uri'] = str(ground_truth_validation.outputs['output_manifest_location'])
+    channelObjList.append(copy.deepcopy(channelObj))
```

Comment:
Can we provide a little helper method (with input parameters for every field) that other users could directly copy to create a new channel?

```
def create_sagemaker_channel(channel_name, s3_uri, s3_data_type='<sane default>', ...):
```

Reply:
If we have a function then they need to specify all the parameters every time they create channel object 
and these inputs can be different so we can't have one default 
[channel_1](https://github.com/akartsky/pipelines/blob/jsonDumps/samples/contrib/aws-samples/mnist-kmeans-sagemaker/mnist-classification-pipeline.py#L20)
[channel_2](https://github.com/akartsky/pipelines/blob/jsonDumps/samples/contrib/aws-samples/ground_truth_pipeline_demo/mini-image-classification-pipeline.py#L16)

with my approach they can just copy the python channel object and modify however they want